### ISSUE TITLE
[Follow-Up] Cross-Lingual Inference Loss After Fine-Tuning On A New Language

### ISSUE THREAD

**Issue by justinjohn0306** [RVC-Boss/GPT-SoVITS#1655]
@RVC-Boss 

This is a follow-up ticket to the issue: [#1626](https://github.com/RVC-Boss/GPT-SoVITS/issues/1626).

### Steps to Reproduce:
1. Fine-tune the base model using a 24-hour Spanish dataset.
2. Align the fine-tuned model with the base.
3. Try cross-lingual inference with other languages (e.g., English, Chinese).

### Expected Behavior:
The model should retain cross-lingual abilities and be able to infer in languages other than Spanish (like English, Chinese, etc.).

### Actual Behavior:
The model appears to have lost the ability to handle languages outside of the one it was fine-tuned on.

### Question:
Is there a recommended way to preserve cross-lingual capabilities during fine-tuning? Should earlier layers be frozen during fine-tuning to avoid losing this ability?

**Comment by RVC-Boss** [comment]
You should fine tune the model using all languages you need to cross when cross-lingual inference.

Freeze earlier layers: It may work, but I haven't do the experiment, and therefore I cann't give you a clear conclusion.

**Comment by justinjohn0306** [comment]
Gotcha, yeah that definitely makes sense!

**Comment by Sheldonimo** [comment]
@justinjohn0306 any update? I am trying to do the same.

**Comment by AngelGuevara7** [comment]
Hey!
I'm trying to do the same too. Could you share some details about the training? epochs and some other training parameters.
I was successful training the SoVITS part but had trouble with the GPT part. When I start training it just prints the dataset batches to the terminal without finishing or throwing errors. Did you experience anything like this?
I leave an example of the behavior observed in the terminal: 'audio_76410.wav': ['e l   t ˌ a β e ɾ n ˈ e ɾ o   k o x j ˈ o   ˈ u n a   k ˈ o p a   ð e   m e t ˈ a l   k e   ɾ ɾ ˌ e l u θ ˈ i a   ð e   t ˈ a n t o   l i m p j ˈ a ɾ l a .  ', 'None', 'El tabernero cogió una copa de metal que relucía de tanto limpiarla.']
Thanks in advance! :)

**Comment by AngelGuevara7** [comment]
Hey @justinjohn0306!
I'm trying to do the same too. I have started with just the Spanish dataset to check if it learns new languages ​​and I already have the training up and running.
I saw in other issues that you already have a Spanish model. Could you share the details of your training params to compare results? Configuration files are fine if you don't mind.
Thanks in advance!!

**Comment by jmdBB** [comment]
Hey! Guys, how long takes training the model? I also need Spanish language available, and how much VRAM does it needs? Does anyone successfully trained and tested? Does it works? 
Thank you !!
Jorge
